akka-actor/src/main/scala/akka/actor/ActorCell.scala

```diff
@@ -17,6 +17,7 @@ import scala.concurrent.duration.Duration
 import java.util.concurrent.ThreadLocalRandom
 import scala.util.control.NonFatal
 import akka.dispatch.MessageDispatcher
+import akka.util.Reflect.lookupAndSetField
```

Comment:
Can we change the actual lines using this method to `Reflect.lookupAndSetField`?
It's more explicit then that where we're calling into then